samples/display/lvgl/boards/native_posix.conf

```diff
@@ -4,3 +4,7 @@ CONFIG_SDL_DISPLAY_Y_RES=240
 
 CONFIG_KSCAN=y
 CONFIG_LVGL_POINTER_KSCAN=y
+
+CONFIG_LVGL_HOR_RES=320
```

Comment:
It relied on defaults before (320x240) which need to match with SDL. It won’t work otherwise, please check pulling this PR if you want.

Reply:
Actually I would assume the defaults(480, 320) should work for native posix as the allocated memory buffer is bigger than the required buffer for the default native_posix resolution 320x240.

Reply:
Yes, this is what I expected too. But I get a black screen:

![image](https://user-images.githubusercontent.com/25011557/87010189-8064da80-c1c6-11ea-9fc0-3ba1022ece6e.png)

Reply:
Looks like the issue is caused by a bug in the LVGL glue logic in case a static rendering buffer is used, so no need in setting these.

See https://github.com/zephyrproject-rtos/zephyr/pull/26827 for the fix